### PR TITLE
Added Birthday Countdown

### LINKED ISSUE #50: Birthday countdown page
## Issue Type:

- [ ] Bug Report
- [x] Feature Request
- [ ] Documentaion

## **About the Project**

- I would like to add a Birthday Countdown webpage which is made using JavaScript, HTML and CSS. 
- The webpage asks for the birthdate input from the user and then accordingly shows a countdown timer.

Kindly assign me this issue!

### PR DESCRIPTION
**Pull request by Dhruv-194**
# Description

- Added a Birthday Countdown webpage made using JavaScript, HTML and CSS. 
- The webpage asks the user to enter his/her birthday date and then shows the countdown accordingly. 

Fixes #50 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Explain the Testing instructions

**Test Configuration**:

- Operating system: Windows 10
- Version:
- Text-editors used: VSCode

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
Screenshot (211)

### CODE REVIEW

**Comment by arpit456jain** [comment]
@Dhruv-194 when I am entring my DOB 15th dec 2000 as 12/15/2000 its not working but when I am entering 12/15/2021 its working

@ankitapuri can you please confirm

**Comment by Dhruv-194** [comment]
@arpit456jain It is meant to be and work like that. A countdown is a timer for the future event and not for an event in the past. 
When you enter a date of past (as in your case) it greets you by saying that 'Hope you have a Nice Birthday' which indicates that event has already occurred and the program is working fine.

**Comment by ankitapuri** [comment]
> @Dhruv-194 when I am entring my DOB 15th dec 2000 as 12/15/2000 its not working but when I am entering 12/15/2021 its working
> 
> @ankitapuri can you please confirm

Yes ! Its showing same to me but as mentioned by @Dhruv-194's understanding its fine then

**Comment by arpit456jain** [comment]
yeah @Dhruv-194 but its asking birth date and i think its little bit odd to enter our birthday can you do something like if we enter our DOB it will show timer
@ankitapuri your views ?

File: Birthday Countdown/style.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  @keyframes gradient {
+    0% {
+        background-position: 0% 50%;
+    }
+    50% {
+        background-position: 100% 50%;
+    }
+    100% {
+        background-position: 0% 50%;
+    }
```
Comment (ankitapuri) [inline Birthday Countdown/style.css:21]: Add 1 more closing bracket

**Comment by Dhruv-194** [comment]
@arpit456jain what I can do is I can change the content of the alert box and ask the user to enter their birthday date. Will that be okay?

**Review by ankitapuri — CHANGES_REQUESTED** [review]

**Comment by arpit456jain** [comment]
> @arpit456jain what I can do is I can change the content of the alert box and ask the user to enter their birthday date. Will that be okay?

yeah that would be great

**Comment by Dhruv-194** [comment]
@arpit456jain @ankitapuri now check. I have updated the files and now it should work as accepted.

**Comment by arpit456jain** [comment]
works fine for me @ankitapuri confirm please

File: Birthday Countdown/style.css (hunk 1 above)
Comment (arpit456jain) [reply Birthday Countdown/style.css]: @Dhruv-194 did you make this change ?

Comment (Dhruv-194) [reply Birthday Countdown/style.css]: Yup I did!

**Review by arpit456jain — APPROVED** [review]

**Comment by arpit456jain** [comment]
great work @Dhruv-194

**Pull request merged**